imgui.cpp

```diff
@@ -5007,6 +5011,36 @@ void ImGui::TextDisabled(const char* fmt, ...)
     va_end(args);
 }
 
+void ImGui::TextHoveredV(const char* fmt, va_list args)
+{
+    ImGui::PushStyleColor(ImGuiCol_Text, GImGui->Style.Colors[ImGuiCol_TextHovered]);
+    TextV(fmt, args);
+    ImGui::PopStyleColor();
+}
+
+void ImGui::TextHovered(const char* fmt, ...)
+{
+    va_list args;
+    va_start(args, fmt);
+    TextDisabledV(fmt, args);
```

Comment:
Want me to remove them and make another pull-request ?
(I'm not very familiar with github...)